docs/develop/node/running-a-node.md

```diff
@@ -188,3 +188,54 @@ To check logs call: `nearup logs` or `nearup logs --follow`
 >Got a question?
 <a href="https://stackoverflow.com/questions/tagged/nearprotocol">
   <h8>Ask it on StackOverflow!</h8></a>
+
+## Starting a node from backup
+There are public backups of data available as tar archives.
+This enables you to sync data from the network faster.
+We have regular backups and archival backups.
+We also keep those for two chains, mainnet and testnet.
+
+### Archive links
+
+Mainnet regular data
+
+https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/rpc/data.tar
+
+Mainnet archival data
+
+https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/archive/data.tar
+
+Testnet regular data
+
+https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/rpc/data.tar
+
+Testnet archival data
+
+https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/archive/data.tar
+
```

Comment:
```suggestion

### Starting the node
```

Reply:
Solved most of the changes, lost context for this one, could you review this PR again

Reply:
lol... not sure what this is either... disregard :)